sdk/python/packages/flet-cli/src/flet_cli/commands/build.py

```diff
@@ -897,20 +1200,15 @@ def ignore_build_output(path, files):
 
             self.cleanup(
                 0,
-                message=f"Successfully built your [cyan]{self.platforms[target_platform]['status_text']}[/cyan]! {self.emojis['success']} "
-                f"Find it in [cyan]{rel_out_dir}[/cyan] directory. {self.emojis['directory']}",
+                message=(
+                    f"Successfully built your [cyan]{self.platforms[target_platform]['status_text']}[/cyan]! {self.emojis['success']} "
+                    f"Find it in [cyan]{rel_out_dir}[/cyan] directory. {self.emojis['directory']}"
+                    f"\nRun [cyan]python -m http.server --directory {rel_out_dir}[/cyan] command to start dev web server with your app. "
+                    if target_platform == "web"
+                    else ""
+                ),
```

Comment:
If the target platform is not web, then `""` will be used as message. We need to condition only the last line of the multiline string:
```suggestion
                message=(
                    f"Successfully built your [cyan]{self.platforms[target_platform]['status_text']}[/cyan]! {self.emojis['success']} "
                    f"Find it in [cyan]{rel_out_dir}[/cyan] directory. {self.emojis['directory']}"
                    + (f"\nRun [cyan]python -m http.server --directory {rel_out_dir}[/cyan] command to start dev web server with your app. "
                    if target_platform == "web"
                    else "")
                ),
```